Marlin/Marlin_main.cpp

```diff
@@ -73,6 +73,9 @@
  *
  * M0   - Unconditional stop - Wait for user to press a button on the LCD (Only if ULTRA_LCD is enabled)
  * M1   - Same as M0
+ * M3   - Turn spindle on, set spindle speed, set rotation to clockwise
+ * M4   - Turn spindle on, set spindle speed, set rotation to counter-clockwise
+ * M5   - Turn spindle off
```

Comment:
Please remove all spindle-related changes from this PR and make them in your `M3`/`M4`/`M5` branch instead.